include/nccl-headers/neuron/error.h

```diff
@@ -6,7 +6,7 @@
 #ifndef NCCL_HEADERS_NEURON_ERROR_H_
```

Comment:
we don't need `stack-info: PR: https://github.com/aws/aws-ofi-nccl/pull/554, branch: aws-nslick/stack/1` in the commit message.

Reply:
This is handled by tooling. After approval `stack-pr land` force pushes an amended commit message, removing this metadata, then merges it. See https://github.com/modularml/stack-pr and [an example pull request on the stack-pr repo](https://github.com/modularml/stack-pr/pull/21)